kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt

```diff
@@ -472,6 +476,19 @@ interface ViewPagerAssertions : BaseAssertions {
 }
 
 /**
+ * Provides assertios for RatingBar
+ */
+interface RatingBarAssertions : BaseAssertions {
+    /**
+     *  Checks If RatingBar has number of rating as expected
+     *  @param number rating as expected
```

Comment:
You should add empty line in between, so that Dokka can generate valid documentation